frontend/ui/wikipedia.lua

```diff
@@ -768,9 +768,9 @@ time, abbr, sup {
 
     -- Done with adding files
     if cancelled then
-        progress_callback(_("Cleaning up..."))
+        progress_callback(_("Cleaning up…"))
```

Comment:
They looked fine in your first patch. You must have changed the whole file encoding to keep my badly encoded url.
So better to forget my bad stuff, and do like you did at first. My other comments should be enough.